### PR TITLE
Feature/bcs federated apiserver 3

### PR DESCRIPTION
**Pull request by wenxinlee2015**
Change apis && client from kubernetes to bcs-federated-apiserver, because kube-builder cannot deal with the apiserver resource automately (Get return list && AddToscheme)

### CODE REVIEW

File: bcs-k8s/bcs-federated-apiserver/pkg/bcs-storage/http.go [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+import (
+	"encoding/base64"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"k8s.io/klog"
+	"net/http"
+)
+
+type Response struct {
```
Comment (abstractmj) [inline bcs-k8s/bcs-federated-apiserver/pkg/bcs-storage/http.go:25]: 导出的结构或者函数最好加上注释
如
// BcsStorage is xxxxxxx

File: bcs-k8s/bcs-federated-apiserver/pkg/configuration/cluster.go [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+	"k8s.io/klog"
+	fedv1b1 "sigs.k8s.io/kubefed/pkg/apis/core/v1beta1"
+	"sigs.k8s.io/kubefed/pkg/client/generic"
+)
+
+type AggregationClusterInfo struct {
+	memberClusterList string
+}
+
+//var ClusterInfo AggregationClusterInfo
```
Comment (abstractmj) [inline bcs-k8s/bcs-federated-apiserver/pkg/configuration/cluster.go:34]: rm it

File: bcs-k8s/bcs-federated-apiserver/pkg/kubectl-agg/pod.go [hunk 3]
```diff
@@ -0,0 +1,280 @@
...
+	"github.com/Tencent/bk-bcs/bcs-k8s/bcs-federated-apiserver/pkg/client/clientset_generated/clientset"
+
+	"k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/klog"
+)
+
+type AggPodOptions struct {
```
Comment (abstractmj) [inline bcs-k8s/bcs-federated-apiserver/pkg/kubectl-agg/pod.go:36]: comments

File: bcs-k8s/bcs-federated-apiserver/pkg/kubectl-agg/pod.go [hunk 4]
```diff
@@ -0,0 +1,280 @@
...
+	ResourceName  string
+	Namespace     string
+	Selector      string
+	AllNamespaces bool
+	WideMessage   string
+	LabelsMessage bool
+	HelpMessage   bool
+}
+
+func GetPodRestartCount(pod v1alpha1.PodAggregation) int32 {
```
Comment (abstractmj) [inline bcs-k8s/bcs-federated-apiserver/pkg/kubectl-agg/pod.go:46]: comment

File: bcs-k8s/bcs-federated-apiserver/pkg/bcs-storage/http.go (hunk 1 above)
Comment (wenxinlee2015) [reply bcs-k8s/bcs-federated-apiserver/pkg/bcs-storage/http.go:25]: 1

File: bcs-k8s/bcs-federated-apiserver/pkg/kubectl-agg/pod.go (hunk 3 above)
Comment (wenxinlee2015) [reply bcs-k8s/bcs-federated-apiserver/pkg/kubectl-agg/pod.go:32]: yeah, has done.

File: bcs-k8s/bcs-federated-apiserver/pkg/kubectl-agg/pod.go (hunk 4 above)
Comment (wenxinlee2015) [reply bcs-k8s/bcs-federated-apiserver/pkg/kubectl-agg/pod.go:43]: yeah, fixed

File: bcs-k8s/bcs-federated-apiserver/pkg/configuration/cluster.go (hunk 2 above)
Comment (wenxinlee2015) [reply bcs-k8s/bcs-federated-apiserver/pkg/configuration/cluster.go]: done

**Review by abstractmj — APPROVED** [review]
LGTM

**Review by DeveloperJim — APPROVED** [review]
LGTM

**Pull request merged**